src/python/pants/option/arg_splitter_test.py

```diff
@@ -98,140 +86,259 @@ def assert_not_positional(arg):
     assert_not_positional('foo')
     assert_not_positional('a_b_c')
 
-  def test_basic_arg_splitting(self):
+  def test_basic_arg_splitting(self) -> None:
```

Comment:
This function is the same, modulo using keyword args and some formatting tweaks.